.github/workflows/ci.yml

```diff
@@ -12,6 +12,6 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: "14.x"
+          node-version: "18.x"
```

Comment:
by updating to Node 18, can we run into issues updating braintree-web to whatever version of asset-loader includes this dependency bump?

Reply:
🤦 disregard my earlier Q, this is the CI YAML file oye I can't read